src/datta_lab_to_nwb/markowitz_gillis_nature_2023/convert_session.py

```diff
@@ -15,94 +15,100 @@
 def session_to_nwb(
     session_id: str,
     data_path: Union[str, Path],
-    optoda_path: Union[str, Path],
-    metadata_path: Union[str, Path],
     output_dir_path: Union[str, Path],
+    experiment_type: str,
```

Comment:
Would recommend this be more precise as a `Literal`

E.g., `experiment_type: Literal["<experiment_name_1>", "<experiment_name_2>"]